relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -156,7 +160,7 @@ pub(crate) fn extract_span_tags(event: &mut Event, config: &Config) {
             continue;
         };
 
-        let tags = extract_tags(span, config, ttid, ttfd, is_mobile);
+        let tags = extract_tags(span, config, ttid, ttfd, is_mobile, start_type.clone());
```

Comment:
```suggestion
        let tags = extract_tags(span, config, ttid, ttfd, is_mobile, start_type);
```

see suggestions below.